### ISSUE TITLE
TS Server fatal error:  Cannot read property 'indexOf' of undefined

### ISSUE THREAD

**Issue by snaplesec** [microsoft/vscode#116344]
Issue Type: <b>Bug</b>

❗️❗️❗️ Please fill in the sections below to help us diagnose the issue ❗️❗️❗️

**TypeScript Version:** 4.1.4

**Steps to reproduce crash**
1.
2.
3.

**TS Server Log**
❗️Server logging disabled. To help us fix crashes like this, please enable logging by setting:
```json
"typescript.tsserver.log": "verbose"
```
After enabling this setting, future crash reports will include the server log.

**TS Server Error Stack**
Server: `semantic`
```
TypeError: Cannot read property 'indexOf' of undefined
    at Object.parsePackageName (c:\Users\SteveNaples\OWL\vite-start\insuredportal-vue-application\node_modules\typescript\lib\tsserver.js:39472:30)
    at ConfiguredProject.Project.enablePlugin (c:\Users\SteveNaples\OWL\vite-start\insuredportal-vue-application\node_modules\typescript\lib\tsserver.js:151962:24)
    at ConfiguredProject.enablePluginsWithOptions (c:\Users\SteveNaples\OWL\vite-start\insuredportal-vue-application\node_modules\typescript\lib\tsserver.js:152477:30)
    at ProjectService.loadConfiguredProject (c:\Users\SteveNaples\OWL\vite-start\insuredportal-vue-application\node_modules\typescript\lib\tsserver.js:154206:25)
    at ProjectService.createAndLoadConfiguredProject (c:\Users\SteveNaples\OWL\vite-start\insuredportal-vue-application\node_modules\typescript\lib\tsserver.js:154145:22)
    at ProjectService.createLoadAndUpdateConfiguredProject (c:\Users\SteveNaples\OWL\vite-start\insuredportal-vue-application\node_modules\typescript\lib\tsserver.js:154150:36)
    at ProjectService.assignProjectToOpenedScriptInfo (c:\Users\SteveNaples\OWL\vite-start\insuredportal-vue-application\node_modules\typescript\lib\tsserver.js:155055:44)
    at c:\Users\SteveNaples\OWL\vite-start\insuredportal-vue-application\node_modules\typescript\lib\tsserver.js:155395:97
    at Object.flatMap (c:\Users\SteveNaples\OWL\vite-start\insuredportal-vue-application\node_modules\typescript\lib\tsserver.js:513:25)
    at ProjectService.applyChangesInOpenFiles (c:\Users\SteveNaples\OWL\vite-start\insuredportal-vue-application\node_modules\typescript\lib\tsserver.js:155395:41)
    at Session.handlers.ts.Map.ts.getEntries._a.<computed> (c:\Users\SteveNaples\OWL\vite-start\insuredportal-vue-application\node_modules\typescript\lib\tsserver.js:156413:46)
    at c:\Users\SteveNaples\OWL\vite-start\insuredportal-vue-application\node_modules\typescript\lib\tsserver.js:158286:88
    at IOSession.Session.executeWithRequestId (c:\Users\SteveNaples\OWL\vite-start\insuredportal-vue-application\node_modules\typescript\lib\tsserver.js:158277:28)
    at IOSession.Session.executeCommand (c:\Users\SteveNaples\OWL\vite-start\insuredportal-vue-application\node_modules\typescript\lib\tsserver.js:158286:33)
    at IOSession.Session.onMessage (c:\Users\SteveNaples\OWL\vite-start\insuredportal-vue-application\node_modules\typescript\lib\tsserver.js:158310:35)
    at Interface.<anonymous> (c:\Users\SteveNaples\OWL\vite-start\insuredportal-vue-application\node_modules\typescript\lib\tsserver.js:160520:27)
    at Interface.emit (events.js:315:20)
    at Interface._onLine (readline.js:329:10)
    at Interface._normalWrite (readline.js:474:12)
    at Socket.ondata (readline.js:186:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    at readableAddChunk (_stream_readable.js:271:9)
    at Socket.Readable.push (_stream_readable.js:212:10)
    at Pipe.onStreamRead (internal/stream_base_commons.js:186:23)
```

VS Code version: Code 1.53.1 (5d424b828ada08e1eb9f95d6cb41120234ef57c7, 2021-02-08T23:29:42.785Z)
OS version: Windows_NT x64 10.0.18363

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-6440HQ CPU @ 2.60GHz (4 x 2592)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|15.89GB (2.52GB free)|
|Process Argv|--crash-reporter-id 0646d7a4-62da-4680-85f9-d9bb72edc6ea|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (35)</summary>

Extension|Author (truncated)|Version
---|---|---
Bookmarks|ale|12.1.4
vscode-apollo|apo|1.18.0
npm-intellisense|chr|1.3.1
bracket-pair-colorizer|Coe|1.0.61
vscode-eslint|dba|2.1.14
githistory|don|0.6.14
console-log|dot|0.6.2
gitlens|eam|11.2.1
prettier-vscode|esb|5.9.1
vscode-pull-request-github|Git|0.23.0
vscode-graphql|Gra|0.3.15
volar|joh|0.20.8
vscode-graphql|luy|0.0.4
vscode-docker|ms-|1.9.1
remote-wsl|ms-|0.53.4
vetur|oct|0.32.0
material-icon-theme|PKi|4.5.0
java|red|0.74.0
vue-vscode-extensionpack|sdr|0.2.0
vue-vscode-snippets|sdr|2.1.5
sonarlint-vscode|Son|1.20.1
vscode-status-bar-format-toggle|tom|2.0.0
vscodeintellicode|Vis|1.2.11
veracode-greenlight-for-vs-code|vs-|1.3.1996016
vscode-java-debug|vsc|0.31.0
vscode-java-dependency|vsc|0.18.0
vscode-java-pack|vsc|0.12.1
vscode-java-test|vsc|0.28.0
vscode-maven|vsc|0.27.1
vscode-jumpy|wma|0.3.1
JavaScriptSnippets|xab|1.8.0
local-history|xyz|1.8.1
preview-insiders|znc|2021.1.21065823
vue|znc|0.6.3
vue-language-features|znc|0.6.3

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
pythonvspyt617cf:30253240
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492:30211401
vstry914:30250197
pythonvsdeb440:30248342
pythonvsded773:30248341
vstes627:30244334
pythonvspyt875:30251590

```

</details>

**Comment by archiekd** [comment]
@snaplesec Same as my issue #116330 

This just happened to me today.

**Comment by archiekd** [comment]
@snaplesec

I found a temp fix here: https://github.com/microsoft/vscode/issues/116219#issuecomment-776407990 

More info in this issue: #116219

**Comment by gjsjohnmurray** [comment]
/duplicate of #116219 and will be fixed in upcoming 1.53.2
